introduction.md

```diff
@@ -5,4 +5,8 @@ I hope to learn something new everyday and fix myself up into the technical worl
 in a smooth easy and correct way.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?